test/remote/gateways/remote_usa_epay_transaction_test.rb

```diff
@@ -22,6 +22,20 @@ def test_successful_purchase_with_track_data
     assert_success response
   end
 
+  def test_successful_authorization_with_manual_entry
+    @credit_card.manual_entry = true
```

Comment:
You're setting `manual_entry` on `@credit_card`, but the setup method is setting up `@creditcard`. Does this test pass? Should it?